server/models/data/availabilitydetails.js

```diff
@@ -0,0 +1,19 @@
+const moment = require('moment');
+
+class AvailabilityDetails {
+
+    constructor(availabilityData) {
+        this.schoolId = availabilityData.schoolId_id;
+        // At any given time, either the worker or student will be null.
+       // this.worker = availabilityData.worker_id,
+        this.workerId= availabilityData.worker_id,
```

Comment:
Hi Nancy, 
Based on the stub, I would also recommend adding `workerTimeslotId` and `date` to this object as they should also be returned in the response.

Reply:
Hi Amy,
Thank you for suggesting that!

My action: I've added `workerTimeslotId` and `date` to the `availabilityData`, by leveraging server/migrations/009.do.create-worker-timeslot-table.sql and server/migrations/013.do.add-create-worker-timeslot-table-data.sql

You can now see these changes in lines 9-10 of server/models/db/availability.js

Please confirm if this takes care of your suggestion!